docs-devsite/auth.actioncodesettings.md

```diff
@@ -46,6 +47,11 @@ android?: {
 
 ## ActionCodeSettings.dynamicLinkDomain
 
+> Warning: This API is now obsolete.
+> 
+> Firebase Dynamic Links is deprecated and will be shut down as early as August 2025. Instead, use  to set a custom domain for mobile links. Learn more in the [Dynamic Links deprecation FAQ](https://firebase.google.com/support/dynamic-links-faq)<!-- -->.
```

Comment:
looks like the `linkDomain` didn't render here in the second sentence.

Reply:
Fixed.